### PR TITLE
Non intuitive "Close currently opened file" dialog text

### PR DESCRIPTION
**Pull request by ybznek**
https://github.com/OpenSCAP/scap-workbench/issues/25

### CODE REVIEW

**Comment by ybznek** [comment]
First commit merge two duplicate codes. Second commit is coming.

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -1406,3 +1397,11 @@ void MainWindow::aboutQt()
 {
     QMessageBox::aboutQt(this);
 }
+
+
+QMessageBox::StandardButton MainWindow::openNewFileQuestionDialog(QString oldFilepath) {
```
Comment (mpreisler) [inline src/MainWindow.cpp]: code style...

File: src/MainWindow.cpp [hunk 2]
```diff
@@ -1406,3 +1397,11 @@ void MainWindow::aboutQt()
...
+    return QMessageBox::question(this, QObject::tr("Close currently opened file?"),
+                               QObject::tr("Opened file '%1' has to be closed before opening another file.\n\n"
```
Comment (mpreisler) [inline src/MainWindow.cpp]: strange indentation

File: src/MainWindow.cpp (hunk 1 above)
Comment (ybznek) [inline src/MainWindow.cpp]: specifically what?

Comment (mpreisler) [inline src/MainWindow.cpp]: code style - newline before scope opening brace
pass by value - the `QString oldFilepath`

Comment (ybznek) [inline src/MainWindow.cpp]: Ah, ok.
QString has COW.. But passing as const & could be faster.

File: include/MainWindow.h [hunk 3]
```diff
@@ -263,6 +264,13 @@ class MainWindow : public QMainWindow
          */
         unsigned int getSelectedRulesCount();
 
+        /**
+         * @brief Ask user to proceed with closing old file due to openning of new one
+         *
+         * @return user answer
+         */
+        QMessageBox::StandardButton openNewFileQuestionDialog(const QString &oldFilepath);
```
Comment (mpreisler) [inline include/MainWindow.h]: The SCAP Workbench codebase uses the `const T& var` convention instead of `const T &var`.

**Pull request merged**